README.md

```diff
@@ -36,6 +36,8 @@ You can configure the behaviour and output of the `tldr` client by setting envir
     export TLDR_COLOR_PARAMETER="white"
     export TLDR_CACHE_ENABLED=1
     export TLDR_CACHE_MAX_AGE=720
+    export TLDR_REMOTE_SOURCE="https://raw.githubusercontent.com/tldr-pages/tldr/master/pages"
```

Comment:
We should name it similar to the flag. If the flag is `--source`, this should be "TLDR_SOURCE".

Reply:
done